lib/mayaUsd/fileio/jobs/jobArgs.cpp

```diff
@@ -1171,6 +1183,35 @@ std::string UsdMayaJobExportArgs::GetResolvedFileName() const
     return file;
 }
 
+static bool isExcluded(const UsdMayaJobExportArgs& args, const TfToken* tokens, size_t count)
+{
+    for (size_t i = 0; i < count; ++i)
+        if (args.excludeExportTypes.count(tokens[i]) > 0)
+            return true;
+    return false;
+}
+
+bool UsdMayaJobExportArgs::isExportingMeshes() const
```

Comment:
These are functions with code taken from existing code and centralized here to avoid code duplication.